src/game/protobuf_include.vpc

```diff
@@ -12,7 +12,7 @@ $Project
 	{
 		$Libexternal $SRCDIR\thirdparty\protobuf-2.6.1\bin\osx32\libc++\libprotobuf [$OSX32]
 		$Libexternal $SRCDIR\thirdparty\protobuf-2.6.1\bin\win32\2013\staticcrt\release\libprotobuf [$WIN32]
-		$Libexternal $SRCDIR\thirdparty\protobuf-2.6.1\bin\linux32\libprotobuf [$LINUX32]
+		$Libexternal $SRCDIR\thirdparty\protobuf-2.6.1\src\.libs\libprotobuf [$LINUX32]
```

Comment:
What's this change?

Reply:
This is where autotools puts build outputs, the previous path is specific to valve builds. Doing this is just easier, and is how thirdparty deps are referenced in a couple of other places, so it's more consistent.